code/src/components/HappyThoughts.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react"
+import GetThoughts from "./GetThoughts"
+import PostThoughts from "./PostThoughts"
+// import {formatRelative } from 'date-fns'
+
+const API_THOUGHTS = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
+const API_LIKES = (thoughtId) =>
+  `https://happy-thoughts-technigo.herokuapp.com/thoughts/${thoughtId}/like`
+
+
+const HappyThougts = () => {
+  /* GET THOUGHTS AND LIKES */
+  const [thougts, setThougts] = useState([])
+
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
+
+  const fetchThoughts = () => {
+    fetch(API_THOUGHTS)
+      .then((response) => response.json())
+      .then((json) => setThougts(json))
+  }
+
+  const postLike = (thoughtId) => {
+    const option = { method: "POST" }
+
+    fetch(API_LIKES(thoughtId), option)
+      .then((response) => response.json())
+      .then(() => fetchThoughts())
+  }
+
+  /* POST THOUGHTS */
+
+  const [newThought, setNewThought] = useState("")
+
+  const handleNewThoughtChange = (event) => {
+    setNewThought(event.target.value)
+  }
+
+  const onFormSubmit = (event) => {
```

Comment:
Easy to follow the code 'cause of good selfexplanatory naming of the consts etc :)